### ISSUE TITLE
getMenuShortcutKeyMaskEx bug

### ISSUE THREAD

**Issue by komomon** [Quitten/Autorize#108]
![image](https://user-images.githubusercontent.com/52700174/222325670-9c552907-6195-4495-8cb1-873e51df16f0.png)

**Comment by timeleaper1999** [comment]
Traceback (most recent call last):
  File "C:\Users\xxx\AppData\Roaming\BurpSuite\bapps\f9bbac8c4acf4aefa4d7dc92a991af2f\Autorize.py", line 23, in registerExtenderCallbacks
    initiator.draw_all()
  File "C:\Users\xxx\AppData\Roaming\BurpSuite\bapps\f9bbac8c4acf4aefa4d7dc92a991af2f\helpers\initiator.py", line 58, in draw_all
    tabs.draw()
  File "C:\Users\xxx\AppData\Roaming\BurpSuite\bapps\f9bbac8c4acf4aefa4d7dc92a991af2f\gui\tabs.py", line 89, in draw
    controlR = KeyStroke.getKeyStroke(KeyEvent.VK_R, Toolkit.getDefaultToolkit().getMenuShortcutKeyMaskEx())
AttributeError: 'sun.awt.windows.WToolkit' object has no attribute 'getMenuShortcutKeyMaskEx'

	at org.python.core.Py.AttributeError(Py.java:176)
	at org.python.core.PyObject.noAttributeError(PyObject.java:965)
	at org.python.core.PyObject.object___getattribute__(PyObject.java:3752)
	at org.python.core.PyObject$object___getattribute___exposer.__call__(Unknown Source)
	at org.python.core.Deriveds.__findattr_ex__(Deriveds.java:59)
	at org.python.core.PyObjectDerived.__findattr_ex__(PyObjectDerived.java:1036)
	at org.python.core.PyObject.__getattr__(PyObject.java:957)
	at gui.tabs$py.draw$7(C:\Users\xxx\AppData\Roaming\BurpSuite\bapps\f9bbac8c4acf4aefa4d7dc92a991af2f\gui\tabs.py:170)
	at gui.tabs$py.call_function(C:\Users\xxx\AppData\Roaming\BurpSuite\bapps\f9bbac8c4acf4aefa4d7dc92a991af2f\gui\tabs.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:173)
	at org.python.core.PyBaseCode.call(PyBaseCode.java:134)
	at org.python.core.PyFunction.__call__(PyFunction.java:416)
	at org.python.core.PyMethod.__call__(PyMethod.java:126)
	at helpers.initiator$py.draw_all$4(C:\Users\xxx\AppData\Roaming\BurpSuite\bapps\f9bbac8c4acf4aefa4d7dc92a991af2f\helpers\initiator.py:58)
	at helpers.initiator$py.call_function(C:\Users\xxx\AppData\Roaming\BurpSuite\bapps\f9bbac8c4acf4aefa4d7dc92a991af2f\helpers\initiator.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:173)
	at org.python.core.PyBaseCode.call(PyBaseCode.java:134)
	at org.python.core.PyFunction.__call__(PyFunction.java:416)
	at org.python.core.PyMethod.__call__(PyMethod.java:126)
	at org.python.pycode._pyx4.registerExtenderCallbacks$2(C:\Users\xxx\AppData\Roaming\BurpSuite\bapps\f9bbac8c4acf4aefa4d7dc92a991af2f\Autorize.py:31)
	at org.python.pycode._pyx4.call_function(C:\Users\xxx\AppData\Roaming\BurpSuite\bapps\f9bbac8c4acf4aefa4d7dc92a991af2f\Autorize.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:173)
	at org.python.core.PyBaseCode.call(PyBaseCode.java:306)
	at org.python.core.PyBaseCode.call(PyBaseCode.java:197)
	at org.python.core.PyFunction.__call__(PyFunction.java:485)
	at org.python.core.PyMethod.instancemethod___call__(PyMethod.java:237)
	at org.python.core.PyMethod.__call__(PyMethod.java:228)
	at org.python.core.PyMethod.__call__(PyMethod.java:218)
	at org.python.core.PyMethod.__call__(PyMethod.java:213)
	at org.python.core.PyObject._jcallexc(PyObject.java:3565)
	at org.python.core.PyObject._jcall(PyObject.java:3598)
	at org.python.proxies.__main__$BurpExtender$23.registerExtenderCallbacks(Unknown Source)
	at burp.bw7.run(Unknown Source)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

**Comment by terminalJunki3** [comment]
@komomon what version of JDK are you using. Also what version of jython? Can you try updating your versions to the latest. The error you provided occurs when the code tries to call the "getMenuShortcutKeyMaskEx" method on a "WToolkit" object but the object doesn't have that method. 

This can happen for many reasons. Some I can think of...

1.) If the version of the Java Development Kit (JDK) being used by the code is different from the version that was used to compile the code. 

2.) There is a problem with the installation of the JDK

3.) There is a conflict with other libraries that the code depends on.

In short I think it's your JDK or jython version. Please let me know.

**Comment by timeleaper1999** [comment]
@terminalJunki3 Thanks for answering.
My version of JDK is 1.8 and I can make sure this problem has nothing to do with version of jython because I've tried different versions including the latest one.So I think it's weird.

**Comment by Quitten** [comment]
@timeleaper1999  - can you share more data regarding your OS?

**Comment by Quitten** [comment]
@timeleaper1999 - I didnt solved the issue, but I did handled it.
Pushed https://github.com/Quitten/Autorize/commit/127b99669776612db853c58ef0c1584cc0cca41b, can you please test if this helped?

I decided set CTRL as the default key for shortcuts in case of such exceptions happening (I assume that is related to the OS you are running on, let me know which is it)

**Comment by Quitten** [comment]
Closed, unless someone else will report this
